src/main/kotlin/com/linchpino/core/controller/AccountController.kt

```diff
@@ -256,4 +257,9 @@ class AccountController(
     fun profile(authentication: Authentication): AccountSummary {
         return accountService.profile(authentication)
     }
+
+    @PutMapping("/profile/password")
```

Comment:
what do think about changing the URL to "/profile/change-password"? It completely is my fault for the issue. I changed it to "change password". I apologies!
conceptually changes password actioned by end-user but reset password is the admin magic. if we are on the same page please change it in all methods and text of this issue.